views/components/TextBox.svelte

```diff
@@ -13,7 +13,8 @@
         let text = highlightedText.text;
 
         if (highlightedText.preventLineBreak) {
-          text = text.replace(/ /g, "&nbsp;");
+          // Prevent line break, by replacing all whitespace characters (including thin spaces)
```

Comment:
Isn't it a better solution to use [white-space](https://developer.mozilla.org/en-US/docs/Web/CSS/white-space) to prevent line breaks? It's just an idea, maybe you have to overwrite it.

Reply:
Thanks for the hint! I didn't think of using the `white-space` property. While using the property probably would be "cleaner", regex has the advantage of replacing all whitespace characters. I don't know what the exact idea here was. Regex would remove the possibility to set \n, \t, etc., while the property would preserve that. Maybe this preservation might prove useful in the future?